src/app/query/bitcoin/address/address.hooks.ts

```diff
@@ -1,14 +1,18 @@
 // @ts-nocheck
 import { sumNumbers } from '@app/common/utils';
 import { BTC_DECIMALS } from '@shared/constants';
+import { BitcoinTransaction } from '@app/common/transactions/bitcoin/transaction.models';
 
 import { useGetTransactionsByAddressQuery } from './transactions-by-address.query';
 import { useGetUtxosByAddressQuery } from './utxos-by-address.query';
 
 // TODO: Export when needed and remove ts-nocheck
-// Possibly kill these and use query hook directly
-function useGetTransactionsByAddress(address: string) {
-  return useGetTransactionsByAddressQuery(address).data;
+// Possibly kill these and use query hook directly?
+export function useGetTransactionsByAddress(address: string) {
+  // Returns up to 50 mempool txs and the first 25 confirmed txs
+  return useGetTransactionsByAddressQuery(address).data?.map((tx: BitcoinTransaction) => {
+    return { ...tx, tx_type: 'bitcoin' };
+  });
```

Comment:
Because this now uses a `map` we're already returning a new object reference on every render. Should call the map in a memo.